src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecoratorPipelineTest.java

```diff
@@ -115,4 +115,21 @@ public void envVarDollarSignEscaping() throws Exception {
         r.assertLogContains("outside container: $string$with$dollars", b);
         r.assertLogContains("inside container: $string$with$dollars", b);
     }
+
+    @Test
```

Comment:
Could slap an `@Ignore` on it and make this mergeable as a reminder of what is broken.